docs/tutorials/02a_training_a_quantum_model_on_a_real_dataset.ipynb

```diff
@@ -0,0 +1,991 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "ideal-strategy",
+   "metadata": {},
+   "source": [
+    "# Training a Quantum Model on a Real Dataset\n",
+    "\n",
+    "This tutorial will demonstrate how to train a quantum machine learning model to tackle a classification problem. Previous tutorials have featured small, artificial datasets. Here we will increase the problem complexity by considering a real-life classical dataset. We decided to pick a very well-known – albeit still relatively small – problem: the Iris flower dataset. This dataset even has its own Wikipedia [page](https://en.wikipedia.org/wiki/Iris_flower_data_set). Although the Iris dataset is well known to data scientists, we will briefly introduce it to refresh our memories. For comparison, we'll first train a classical counterpart to the quantum model.\n",
+    "\n",
+    "So, let's get started:\n",
+    "\n",
+    "- First, we'll load the dataset and explore how it looks.\n",
```

Comment:
```suggestion
    "- First, we'll load the dataset and explore what it looks like.\n",
```
I think how it is here now is how it was originally. The how it looks is normally something someone will use when asking for a judgment - as in how does this hat look on me - it makes you look old!  For me here we just want to see what the dataset contains - what it looks like. Hence I would use the text above - which we had at one point. If you really prefer the how - then it does get confounded this way and I think the intent is clear though its not what I'd use.

Reply:
Nice example :). I'd not considered that "how it looks" carries a value judgement. I think somewhere else I'd changed "how it looks like" to "how it looks" and then stuck to that. But I'm happy with "what it looks like".

Reply:
Thanks for the English grammar lesson, appreciated :)